src/components/Button.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
Jeg er kanskje litt i mot å bruke dette navnet siden vi bruker denne i `ndla-ui`. Kunne vi kalt den noe annet? For eksempel `SaveButton`?